ttml/profile/dapt/Overview.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Dubbing and Audio description Profiles of TTML2 :: Namespaces</title>
+  </head>
+
+  <body>
+    <h1></h1>
+
+<p>This is a placeholder for a namespace document that permits dereferencing the Dubbing and Audio description Profiles of TTML2 namespace URIs specified at https://www.w3.org/TR/dapt/ :</p>
+
+<pre>
+http://www.w3.org/ns/ttml/profile/dapt#metadata
+http://www.w3.org/ns/ttml/profile/dapt/extension/
```

Comment:
How will both URIs dereference to this same resource (`Overview.html`)?

Reply:
I'm not sure I could understand what is pointed, but existing ones just list URL as plain text without href link, so I've put these as text.
https://github.com/himorin/w3c-ns/blob/main/ttml/profile/imsc1/Overview.html